lms/djangoapps/experiments/admin.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib import admin
+
+from .models import ExperimentData
+
+
+@admin.register(ExperimentData)
+class ExperimentDataAdmin(admin.ModelAdmin):
```

Comment:
Ok, so we just need to make sure never to look at it in prod? Is this a footgun worth defusing? (Didn't you have a similar one in ecommerce?)